.cirrus.yml

```diff
@@ -333,6 +333,51 @@ task:
       on_failure:
         <<: *on_failure_ac
 
+    - name: Linux - Debian Bullseye - Meson - Test Extensions
+      allow_failures: true
+      depends_on:
+        - Linux - Debian Bullseye - Meson
+
+      configure_script: |
+        su postgres <<-EOF
+          meson setup \
+            --buildtype=debug \
+            --prefix=/usr/local/ \
+            -Dcassert=true \
+            ${LINUX_MESON_FEATURES} \
+            -DPG_TEST_EXTRA="$PG_TEST_EXTRA" \
+            build
+        EOF
+      build_script: su postgres -c 'ninja -C build -j${BUILD_JOBS}'
+
+      install_script: ninja -C build install
+
+      install_required_packages_for_extensions_script: |
+        apt-get update
+        DEBIAN_FRONTEND=noninteractive apt-get -y install \
+          autoconf \
+          libtool \
+          libgeos-dev \
+          libproj-dev \
+          libprotobuf-c-dev \
+          protobuf-c-compiler
+
+      # some of the make commands of test_extensions script needs to run with sudo,
+      # that command makes postgres user can execute sudo commands without password
+      dont_ask_sudo_pw_script: |
+        echo "postgres ALL=(ALL:ALL) NOPASSWD: ALL" | tee /etc/sudoers.d/postgres
+
+      always:
+        test_pg_qualstats_script: su postgres -c 'python3 src/tools/ci/test_extensions --only pg_qualstats'
```

Comment:
Why are you starting the script as postgres, just to then sudo out of the postgres user from within the script?